next.config.js

```diff
@@ -3,6 +3,14 @@ const nextConfig = {
   compiler: {
     styledComponents: true,
   },
+  images: {
+    domains: [
+      "images.unsplash.com",
+      "images.pexels.com",
+      "www.mojapraktika.com",
+      "homegymler.de",
```

Comment:
Please make sure to use only images that are not protected by copyright. You should verify with these two sources that you are allowed to use the images without any restrictions.